### PR TITLE
Adds chef client warning

### PR DESCRIPTION
**Pull request by spuder**

### CODE REVIEW

File: salt/modules/chef.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def __virtual__():
     Only load if chef is installed
     '''
     if not salt.utils.which('chef-client'):
-        return False
+        return (False, 'Cannot load chef module: chef-client not found'
```
Comment (cachedout) [inline salt/modules/chef.py]: I think you may need to add a trailing paren here. Thanks!

Comment (spuder) [inline salt/modules/chef.py]: :neutral_face: derp

**Comment by spuder** [comment]
resubmitted

**Comment by basepi** [comment]
Awesome, thanks @spuder !

**Pull request merged**